Licensing: change the structure (#7097)
DevExpress/devextreme-documentation: concepts/Common/Licensing/00 Licensing.md

### v23.2+

> With the release of v23.2, DevExpress checks license status to determine whether a valid DevExtreme license exists. If you do not have a valid DevExtreme license, an error message will be displayed on-screen. To complete the license verification process, you must [supply a valid license key](/Documentation/Guide/Common/Licensing/#Validate_Your_License_Key). This license key will be auto-generated on your behalf once you purchase a license (it will be auto-generated for all active DevExtreme subscribers as well).

Reviewer: ```suggestion
With the release of v23.2, DevExpress checks the license status to determine whether a valid DevExtreme license exists. If you do not have a valid DevExtreme license, an error message is displayed on-screen. To complete the license verification process, you must [supply a valid license key](/Documentation/Guide/Common/Licensing/#Validate_Your_License_Key). This license key is auto-generated on your behalf once you purchase a license (it is also auto-generated for all active DevExtreme subscribers).
```
